cpp/src/IceSSL/SSLEngine.h

```diff
@@ -26,6 +27,7 @@ namespace IceSSL
 
         Ice::LoggerPtr getLogger() const;
         Ice::PropertiesPtr getProperties() const;
+        Ice::InitializationData getInitializationData() const;
```

Comment:
It's clear to me who is the client for this public exported class.

Is it a public / user-documented API even though there is no doc-comment.

Reply:
We should move this to IceInternal, it is only used by the SSL transport implementation.